target/linux/ramips/dts/mt7620n.dtsi

```diff
@@ -248,6 +248,14 @@
 				ralink,function = "uartlite";
 			};
 		};
+
+                wled_pins: wled {
```

Comment:
Please make sure that all device tree source files including this dtsi and using <&gpio3 0> have the wled group set to gpio. It could be that the bootloader of some boards set the wled pin to gpio and this change would revert this setting and causes regressions.